src/style.css

```
@@ -1,3 +1,59 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  --color: #00acdc;
```

It is a good practice to name colors more descriptively so your code is readable
```suggestion
  --color-blue: #00acdc;
```